src/components/header/search/Search.scss

```diff
@@ -43,10 +43,32 @@
         &:focus {
             text-align: left;
         }
-    }
 
+        @include mini-screen {
+
+            // WebKit browsers
+            &::-webkit-input-placeholder {
+             color:  #fff;
```

Comment:
This sets a white text color to a white background. Is there a reason why? :)

Reply:
This is now changed to transparent instead of white.